kernel/byterun/dune

```diff
@@ -5,6 +5,6 @@
  (c_names coq_fix_code coq_memory coq_values coq_interp))
 
 (rule
- (targets coq_jumptbl.h)
- (deps (:h-file coq_instruct.h) make_jumptbl.sh)
- (action (bash "./make_jumptbl.sh %{h-file} %{targets}")))
+ (targets coq_instruct.h coq_jumptbl.h)
+ (deps (:gen ../genOpcodeFiles.exe))
+ (action (progn (run %{gen} enum) (run %{gen} jump))))
```

Comment:
I assume that `%{gen} enum` produces the first target, while `%{gen} jump` produces the second target. If that's the case, your rule is correct but it's less optimal than doing it in two separate rules. Now dune will fail to parallelize these 2 actions.

Reply:
Thanks for the tip.

Reply:
Yup indeed I was thinking of actually having a single `%{gen} jump gen` call [as to avoid a double process spawn but indeed two rules may be better in the end.